views/layout.njk

```diff
@@ -9,7 +9,8 @@
 <body>
     <!-- Navbar expected on every page -->
     {% include 'navbar.njk' %}
-	{% block content %}{% endblock %}
+	{% block content %}
```

Comment:
Makes no difference.. but why? Other blocks are on one line.

Reply:
Okay, I'll change it back.